core/iwasm/common/wasm_shared_memory.c

```diff
@@ -298,7 +298,7 @@ notify_wait_list(bh_list *wait_list, uint32 count)
 }
 
 static AtomicWaitInfo *
-acquire_wait_info(void *address, bool create)
+acquire_wait_info(void *address, bool create, AtomicWaitNode *wait_node)
```

Comment:
The function is called with either `acquire_wait_info(address, true, wait_node)` or `acquire_wait_info(address, false, NULL)`, seems we can remove argument `bool create`, only keep `wait node`.

Reply:
Fixed